content/linked-terms.js

```diff
@@ -22,5 +22,9 @@ module.exports = [
   {
     matches: 'dvc experiments',
     url: '/doc/command-reference/exp'
+  },
+  {
+    matches: 'dvc ls',
+    url: '/doc/command-reference/list'
```

Comment:
@rogermparent I added `dvc ls` here but it's still not linking. See https://dvc-org-ref-aliases-xeonj8d4vz.herokuapp.com/doc/command-reference/list

Reply:
It works for me locally, so it seems like this is a cache issue.

Reply:
Thanks. Is it possible to destroy the Heroku review app and recreate it from scratch?

Reply:
If you set the `DEPLOY_OPTIONS` env var to `build,upload,clean` it should clear out the cache, though it's possible the same issue could persist in master. If you bump `version` in `plugins/gatsby-remark-dvc-linker`, it should do a cache clear naturally in here as well as master. 
I'm looking into a way to invalidate cache when these settings change, but it'll take some changes on Gatsby's end as well as ours even if we pass options through the proper API.

Reply:
No need to complicate things. But so you think merging this will actually address the problem in question? If so please do @rogermparent !

Reply:
Not quite, I think a proper addressing of the problem will take some extra work. Merging this into master _may_ not exhibit this problem, but it's also possible it could until the next automatic cache clear (most likely a package upgrade).

I think until then, bumping a patch version on `plugin/gatsby-remark-dvc-linker/package.json` is a quick and simple way to solve the problem here and make sure it doesn't happen on master.

Reply:
> bumping a patch version

Done and merged. Let's see what happens... ⌛

Reply:
https://dvc.org/doc/command-reference/list looks good 👍